crates/pop-contracts/src/node.rs

```diff
@@ -137,8 +160,12 @@ pub async fn run_contracts_node(
 	// Wait until the node is ready
 	sleep(STARTUP).await;
 
+	#[cfg(feature = "v5")]
 	let data = Value::from_bytes(subxt::utils::to_hex("initialize contracts node"));
+	#[cfg(feature = "v5")]
 	let payload = subxt::dynamic::tx("System", "remark", [data].to_vec());
+	#[cfg(feature = "v6")]
+	let payload = MapAccount::new().build();
```

Comment:
When running the `ink-node` in development mode, we map Alice by default. This improves the developer experience by allowing quick setup without prompting for a map, and it's also useful for running tests.